tests/data/test_item.json

```diff
@@ -34,6 +34,7 @@
     "type": "Polygon"
   },
   "properties": {
+    "description": "Landat 8 imagery radiometrically calibrated and orthorectified using gound points and Digital Elevation Model (DEM) data to correct relief displacement.",
```

Comment:
free-text for `items` will only work within `properties` (title, description keywords) 

https://github.com/stac-utils/pgstac/blob/45ac2478b58946529872ec3feed0ee0c838c4742/src/pgstac/sql/004_search.sql#L225-L227

Reply:
Not relevant for this PR, but this is why item-level free-text search feels funny to me ... IMO this info should live at the collection level only.